is_core/contrib/background_export/migrations/0001_initial.py

```diff
@@ -27,7 +29,7 @@ class Migration(migrations.Migration):
                 ('slug', models.SlugField(max_length=32, primary_key=True, serialize=False, verbose_name='slug')),
                 ('file', chamber.models.fields.FileField(
                     blank=True, null=True, upload_to=is_core.contrib.background_export.models.generate_filename,
-                    verbose_name='file'
+                    verbose_name='file', storage=import_string(settings.BACKGROUND_EXPORT_STORAGE_CLASS)()
```

Comment:
Tohle se takto vygenerovalo pomocí make-migration?

Reply:
jistě že ne. To jsem napsal ručně.. Stejně jako ti django nevygeneruje migrace s settings.USER_MODEL